certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -1,9 +1,9 @@
 """Module executing integration tests against certbot core."""
 from __future__ import print_function
 
-import os
-from os.path import exists
-from os.path import join
+import os   # pylint: disable=os-module-forbidden
```

Comment:
Here and for the next two lines, the disable directives should not be needed anymore.